projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/class-boost-cache.php

```diff
@@ -329,12 +313,12 @@ public function rebuild_on_comment_post( $comment_id, $comment_approved, $commen
 			 */
 			if ( isset( $parameters['cookies'] ) && ! empty( $parameters['cookies'] ) ) {
 				$filename = trailingslashit( get_permalink( $post->ID ) ) . Filesystem_Utils::get_request_filename( $parameters );
-				$this->storage->invalidate( $filename, Filesystem_Utils::DELETE_FILE );
+				$this->rebuild_page( $filename );
```

Comment:
This is broken atm. It should delete the specific file with specific parameters. This need to be addressed before the PR is merged.

Reply:
PR has been updated. This `rebuild_page` should have been `delete_page` too. It's because the cached page was generated by a visitor with cookies who just left a comment. The cookies will be unique to that visitor.